### PR TITLE
chia: 1.7.0 -> 2.0.1

### PR DESCRIPTION
**Pull request by bbjubjub2494**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by lourkeur** [comment]
I think the python tooling is not checking dependencies...

**Comment by abueide** [comment]
Just curious what the status on this is, let me know if there's anyway I can help! Is it possible we can add an option for the electron gui app to this package or should we add it in a new separate package?

**Comment by chasevasic** [comment]
> Just curious what the status on this is, let me know if there's anyway I can help! Is it possible we can add an option for the electron gui app to this package or should we add it in a new separate package?

I'm not a package maintainer, however I'll mention that before Nix had Chia, I wrote a derivation for personal use. The GUI is both more difficult to package and maintain, as well as having more dependencies many people would not want to pull in (e.g. if they are running Chia on a server, raspberry  pi, headless nodes in general.)  (*cough* electron)  It would be great to have the GUI in nixpkgs. I would say definitely a seperate package.

I'd love to help with Chia too, I've been a Chia supporter since the mainnet launch day. I don't currently know any of the best practices or style rules for nixpkgs, and couldn't currently commit to being a maintainer even if I was more familiar though.

File: pkgs/development/python-modules/chia-rs/default.nix [hunk 1]
```diff
@@ -3,17 +3,17 @@
 , fetchFromGitHub
 , pytestCheckHook
 , rustPlatform
+, clvm-tools
 }:
-
 buildPythonPackage rec {
   pname = "chia-rs";
-  version = "0.2.0";
+  version = "0.2.10";
```
Comment (natsukium) [inline pkgs/development/python-modules/chia-rs/default.nix:10]:
```suggestion
  version = "0.2.10";
  pyproject = true;
```

File: pkgs/development/python-modules/chia-rs/default.nix [hunk 2]
```diff
@@ -39,8 +39,9 @@ buildPythonPackage rec {
     touch wheel/Cargo.lock
   '';
 
-  nativeCheckInputs = [
+  checkInputs = [
```
Comment (natsukium) [inline pkgs/development/python-modules/chia-rs/default.nix:42]:
```suggestion
  nativeCheckInputs = [
```

File: pkgs/development/python-modules/chiapos/default.nix [hunk 3]
```diff
@@ -16,21 +18,27 @@
 
 buildPythonPackage rec {
   pname = "chiapos";
-  version = "1.0.11";
+  version = "2.0.2";
```
Comment (natsukium) [inline pkgs/development/python-modules/chiapos/default.nix:21]:
```suggestion
  version = "2.0.2";
  pyproject = true;

```

**Comment by lourkeur** [comment]
closed in favor of #270254

**Pull request closed without merging**